packages/core/core/src/requests/EntryRequest.js

```diff
@@ -122,6 +122,40 @@ class EntryResolver {
           entries: [{filePath: source, packagePath: entry}],
           files: [{filePath: pkg.filePath}],
         };
+      } else if (pkg && Array.isArray(pkg.source)) {
+        let source;
+        const entries = [];
```

Comment:
nit: `let entries = [];`, even though it's not reassigned this is a style thing in Parcel (though I personally also would prefer const)

Reply:
Ok. Thanks for the review

Reply:
@wbinnssmith I have submitted an updated version of this pull request, kindly review it thanks.

Reply:
looks good! Mind writing an integration test that exercises this code? Should be good to go from there 😄

Reply:
@wbinnssmith I'll do that later today.